### PR TITLE
Milestone1 comments update

### PR DESCRIPTION
**Pull request by Favourolotu**
Applied Feedback from milestone 1:
- [Updated the HTML page names]
- [Added Unit-tests for the user and shopping cart model]
- [Updated the user controller unit tests]
- [Added a link between a shopping cart and a user]

### CODE REVIEW

File: src/test/java/project/persistence/ShoppingCartRepositoryTest.java
Comment (chase-scott) [inline src/test/java/project/persistence/ShoppingCartRepositoryTest.java:1]: These should be deleted

File: src/test/java/project/persistence/UserRepositoryTest.java
Comment (chase-scott) [inline src/test/java/project/persistence/UserRepositoryTest.java:1]: These should be deleted

Comment (Favourolotu) [reply src/test/java/project/persistence/UserRepositoryTest.java:1]: I have them removed

**Review by chase-scott — APPROVED** [review]
Looks good to me! As long as the empty persistence tests dont get added.

**Comment by j-ctrl7** [comment]
The unit tests look good

**Review by j-ctrl7 — APPROVED** [review]
changes look alright to merge

**Pull request merged**